### PR TITLE
chore: post rewrite fixes

### PR DESCRIPTION
**Pull request by savioxavier**
This PR updates/changes the following stuff:
- Updated styles and wording for multiple components
- Changed monospace font to Fira Code
- Fixed code mockup issues
- Fixed some responsiveness issues
- Added two new fonts: Open Sans and Cabin to override system default
	- These two were the best pairings for Montserrat I could find
- Fixed some unnecessary bolding of text
- Formatted and cleaned up some files

### CODE REVIEW

**Comment by savioxavier** [comment]
@megatank58 Can I use FontAwesome to render icons? I know it's a huge library, but it was kind of tough to work with SVGs, and FontAwesome's font implementation is so much easier. Plus there is more than one icon in the site, and more icons might be added too for aesthetic purposes.

**Comment by megatank58** [comment]
Does font awesome only bundle the icons we use? if it doesn't bundles all icons then I'm fine with using it

**Comment by savioxavier** [comment]
I was unable to resolve solid Font Awesome icons for some reason, got the Golang icon ready.

**Comment by megatank58** [comment]
Linter workflow is failing, lint the codebase

File: src/App.vue [hunk 1]
```diff
@@ -1,15 +1,14 @@
 <template>
-  <Navbar />
-  <router-view />
-   <Footer/>
-   <MadeBy/>
+	<Navbar />
+	<router-view />
+	<Footer />
+	<MadeBy />
```
Comment (megatank58) [inline src/App.vue]: MadeBy component isn't imported here

File: src/components/Install.vue [hunk 2]
```diff
@@ -40,16 +41,15 @@ export default defineComponent({
...
+				`	package main
 
 	import "github.com/gominima/minima"
 	
 	func main() {
 		app := minima.New()
 		app.Listen(":3000")
 	}
-			`.trim(),
+			`,
```
Comment (megatank58) [inline src/components/Install.vue:52]: why does `package main` has so many tabs and why was trim removed

**Comment by savioxavier** [comment]
> Linter workflow is failing, lint the codebase

Fixed!

**Review by megatank58 — CHANGES_REQUESTED** [review]

**Comment by megatank58** [comment]
just some things, rest looks good to me

**Comment by apoorvcodes** [comment]
@savioxavier I don't like the fonts, too thin.

**Comment by savioxavier** [comment]
> @savioxavier I don't like the fonts, too thin.

I just selected a few fonts in order to override the system defaults (it doesn't look too good on Android). I couldn't find any other better font combos for Montserrat, so I used Cabin and Open Sans. The font weight is the _default_ font weight, provided by all the fonts, but that could be adjusted later. If you wish, you can suggest or provide other fonts.

The initial font weights which you used for Montserrat (weight 700) struck me as _too bold_, so I resorted to the default weight, meaning to adjust it later.

**Comment by savioxavier** [comment]
@apoorvcodes Deleted the Cabin font, updated font widths to make them bold, and added Pacifico font to render the logo.

You can always add new fonts/change them after this PR is merged

**Review by megatank58 — APPROVED** [review]

**Pull request merged**